app/models/user.rb

```diff
@@ -220,9 +225,23 @@ def sanitize_languages
     filtered_languages.reject!(&:blank?)
   end
 
-  def update_statistics!
+  def prepare_new_user!
     BootstrapTimelineWorker.perform_async(account_id)
     ActivityTracker.increment('activity:accounts:local')
     UserMailer.welcome(self).deliver_later
   end
+
+  def prepare_returning_user!
+    ActivityTracker.record('activity:logins', id)
```

Comment:
Couldn't you track more than once per day if the user logs out and then logs in again?

Reply:
@renatolond It's a weekly hyperloglog. I.e. adding the same id 2000 times won't count as more than 1 per week.

Reply:
Ah, okay, since the previous code was after the return I wasn't sure :)